white-paper/wp.tex

```diff
@@ -174,8 +174,10 @@ \subsubsection{Locators}
 Identities and Network design of IPFS by J. Benet.
 
 \subsubsection{Network}
-DeGitX can use different network protocols for RPC calls, \code{TCP} is supported by default
-\todo{other protocols, e.g. NAT traversing?}.
+DeGitX uses STC, TCP or UDP protocols for transport layer. It uses IPv4 or IPv6 for addressing,
+it stores addresses in \href{https://github.com/multiformats/multiaddr}{multiaddr}, for instance:
```

Comment:
Agree with `multiaddr`, and TCP should be supported by default. I don't known where we can use `STC` for the transport layer, maybe we do not need to support it.

Reply:
@sunchao9 thanks, I've updated this section